src/main/kotlin/io/easybreezy/project/application/issue/command/parser/Normalizer.kt

```diff
@@ -0,0 +1,36 @@
+package io.easybreezy.project.application.issue.command.parser
+
+import com.google.inject.Inject
+import io.easybreezy.project.model.issue.Label
+import io.easybreezy.project.model.issue.Priority
+import java.util.UUID
+
+class Normalizer @Inject constructor(
+    private val categoryConverter: CategoryConverter,
```

Comment:
will be better create converter interface which will be controlled by generic type like CategoryConverter: Converter<List<UUID>?> also we add common function convert(project: UUID, parserIssue: ParsedIssue):  T.
All converters will resolve only self data. Now it's hard to extend converter, we should always add converter on a constructor. When we add interface we'll can build collection of converters